dropwizard-metrics/dropwizard-metrics/src/main/java/com/wavefront/integrations/metrics/WavefrontReporter.java

```diff
@@ -1,27 +1,20 @@
 package com.wavefront.integrations.metrics;
 
+import com.codahale.metrics.*;
```

Comment:
We actually avoid wild card imports.
You can copy system/.idea/codeStyleSettings.xml into java repo and IntelliJ will take care of this.